src/client/sandbox/native-methods.js

```diff
@@ -489,6 +489,9 @@ class NativeMethods {
             this.responseUrlGetter    = win.Object.getOwnPropertyDescriptor(win.Response.prototype, 'url').get;
         }
 
+        if (win.Promise)
```

Comment:
You can remove this condition.
If browser supports `fetch` requests then it means `Promise` is supported too.

Write a separate issue to store `win.Promise.prototy.reject` method and use it in this places: [1](https://github.com/DevExpress/testcafe-hammerhead/pull/1709/files#diff-6a868b3a2dc4967657f6f7dcb7f65f19R116), [2](https://github.com/DevExpress/testcafe-hammerhead/pull/1709/files#diff-6a868b3a2dc4967657f6f7dcb7f65f19R120)